src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.tsx

```diff
@@ -39,6 +48,21 @@ const MigrationPoliciesEmptyState: FC = () => {
       </EmptyStateBody>
       <EmptyStateFooter>
         <EmptyStateActions>
+          {loadError && (
+            <Popover
+              bodyContent={() =>
+                t(
+                  'To perform this action you must get permission from your Organization Administrator.',
+                )
+              }
+              aria-label={'Help'}
+              position={PopoverPosition.right}
+            >
+              <div className="MigrationPoliciesEmptyState__button">
+                <Button>{t('Create MigrationPolicy')}</Button>
+              </div>
+            </Popover>
+          )}
```

Comment:
We should add these changes to the MigrationPoliciesCreateButton as it's also can be rendered if we have items on the list

Reply:
This is problematic again, because the button won't be rendered at all (because of how PF implemented that) in case of the user without permissions, while we do need to render it (because of the design). I would have to add another prop to MigrationPoliciesCreateButton, but Matan told me I won't need that, but then we are not able to achieve the expected look, if I won't put this code in here. Because we need to check loaded or loadError at least. And this seems to me better (while checking the permissions once) than checking permissions of the same thing again and again.

Reply:
Have you tried this? All changes I suggested here was tested locally and works as expected.
Please try this on your branch

Reply:
Depends on what we think about how the expected result looks like...